presto-benchto-benchmarks/src/test/resources/sql/presto/tpcds/q01.plan.txt

```diff
@@ -4,7 +4,7 @@ local exchange (GATHER, SINGLE, [])
             join (LEFT, REPLICATED):
                 join (INNER, PARTITIONED):
                     remote exchange (REPARTITION, HASH, ["c_customer_sk"])
-                        scan tpcds:customer:sf3000.0
```

Comment:
I am not the fan of this. Notice that these plans are generated for particular schema. When you change the schema (scale factor) then generated plans may differ. So I would prefer to have this information stored.

Reply:
> What is the motivation?

The idea is that one can use different connector (e.g: Hive) to validate plans generated with stats from that connector. For that we need a pure table name as each `ConnectorTableHandle` prints differently.

> When you change the schema (scale factor) then generated plans may differ. So I would prefer to have this information stored.

When plans are generated for different schema then I think schema should be part of plan file path, e.g: `test/resources/sql/presto/tpcds/sf100/q01.plan.txt`.

Notice that in benchto queries we also don't use explicit schemas so there is flexibility where queries are run (e.g ORC, parquet, different schemas).